README.md

```diff
@@ -16,6 +16,23 @@ that includes a link to your comment.
 For in-depth discussion of a particularly hairy function, open a dedicated issue
 or pipe up on [the RustDC Zulip chat](https://rust-dc.zulipchat.com/).
 
+Getting Started
+---------------
+
+Cargo and rustc are a prerequisite, to install check https://rustup.rs/.
+
+To build and run tests.
+
+```fish
+> cargo test
```

Comment:
With the `>`, it's not actually valid fish syntax. (Could also leave the `>` but remove the `fish` code-block annotation.

```suggestion
cargo test
```

Reply:
`>` is supposed to be the fish prompt, should we remove that?

Reply:
Yeap (if you're describing valid fish syntax).

Reply:
So I assume this is resolved?

Reply:
Looks like the `>` is still there? Will go ahead and remove them + merge.

Reply:
Ah, I did not know you want me to remove them. Sorry about that.